### PR TITLE
cherry-picked cli removal

### PR DESCRIPTION
**Pull request by isao**
note run.js has some modifications that we'll want to also cherry-pick or copy over later. this pr is the result of this cherry-pick onto `next`, with conflict cleanup

```
git cherry-pick cea8d3db5b137a3f69b2a87766b587b19e364a01 -m 1
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,22 +16,15 @@
         "Lichun Zhan <lzhan@yahoo-inc.com>"
     ],
     "dependencies": {
-        "colors": "*",
         "debug": "*",
+        "express": "3.x",
```
Comment (imalberto) [inline package.json]: Please move back to `devDependencies` as the application will make express its `dependencies`.

File: tests/func/applications/frameworkapp/commandline/package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
               "email": "nobody@yahoo-inc.com"
```
Comment (imalberto) [inline tests/func/applications/frameworkapp/commandline/package.json]: Should commandline/ folder be deleted ?

Comment (isao) [inline tests/func/applications/frameworkapp/commandline/package.json]: yes I noticed `tests/func/applications/frameworkapp/commandline/*` too, and it does look like it's no longer used. will also rm from head of develop.

**Comment by imalberto** [comment]
+1

**Pull request merged**